config/application.rb

```diff
@@ -7,11 +7,23 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+##config.active_record.yaml_column_permitted_classes = [ActiveSupport::HashWithIndifferentAccess]
+##config.active_record.yaml_column_permitted_classes = [Symbol, Hash, Array, ActiveSupport::HashWithIndifferentAccess]
+
 
 module Pb
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 5.2
+    config.load_defaults 7.0
```

Comment:
Amd then again here -- is it possible the version change fixes the load issue? The serialization checks by yaml have a purpose (security) so we should only relax them as little as possible

Reply:
No, just the version change did not fix this issue. I also had to add the change in election.rb to make it work.